interfaces.md

```diff
@@ -6,7 +6,17 @@
 
 # Purpose
 
-To describe flow of the tokens in ODA.
+To describe flow of the tokens in ODA and reference to roles and user definition
+
+# Users
+
+Users are generated in the Drupal web interface, reuired fields are in [the note](https://github.com/oda-hub/doc-multi-user/blob/CF_proposal/user-fields.md)
```

Comment:
I would say users are  defined, though generated is fine if you prefer.
The reference to PR branch will go away, so maybe we can directly use that.